runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -648,6 +635,11 @@ private void takeScreenshot(String fileName, Image image)
 		{
 			File screenshotFile = new File(playerFolder, fileName + ".png");
 
+			for (int offset = 1; screenshotFile.exists(); offset++)
```

Comment:
Consider adding a comment above this block to explain the appending. I naively thought that time would be unique (it's not). Maybe like:

// While time is part of the file name, events like Daylight Savings Time can cause non unique LocalDateTimes to occur. To avoid overwriting files, append _1, _2, etc to the end of the file name.